jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Could you please explain why `data/models` and `data/datasets` are not task env configuration?
How can I distinguish task env configuration from other data?